scenarios/.gitignore

```diff
@@ -1,4 +1,6 @@
 *
 !.gitignore
 !base_2020/
-!mtp_2050/
\ No newline at end of file
+!2030/
+!2040/
+!2050_Amendment/
```

Comment:
Why create a 2050_Amendment? It's still the 2050 scenario right? The amendment is handled by git, so I'd suggest just changing the mtp_2050 folder. Unless you guys specifically want to people to be able to run the old 2050 and the amended. In my mind, this just seems confusing for consultants/public.

Reply:
OK!